src/components/grindery/DexLiquidityWallet/DexLiquidityWallet.tsx

```diff
@@ -61,9 +61,9 @@ const DexLiquidityWallet = (props: Props) => {
         subheader={Object.keys(wallet.tokens)
           .map(
             (key: string) =>
-              `${key}: ${parseFloat(
-                wallet.tokens[key] || '0'
-              ).toLocaleString()}`
+              `${key}: ${parseFloat(wallet.tokens[key] || '0')
+                .toFixed(4)
```

Comment:
@milukove maybe there is a better way to display the amount with decimal placas

Reply:
@milukove when the user withdraws all amount from his wallet. The amount displayed is `0.0000`. 
Should we erase the `token` and `amount` from mongo db to not show 0 values ? 

![image](https://user-images.githubusercontent.com/5060670/227228684-c331524a-5715-47cb-b3df-510378863aa4.png)

Reply:
@juniormp  I don't think we should. We can keep it showing 0, it is just an empty wallet, I see no harm in it.